dspace/config/modules/rest.cfg

```diff
@@ -72,14 +72,6 @@ rest.properties.exposed = spring.servlet.multipart.max-file-size
 rest.properties.exposed = authentication-shibboleth.show.idp-attributes
 rest.properties.exposed = item-page.show-handle-and-doi
 
-# TUL
-rest.properties.exposed = dspace.ui.url
-rest.properties.exposed = versioning.item.history.include.submitter
-rest.properties.exposed = statistics.cache-server.uri
-rest.properties.exposed = citace.pro.url
-rest.properties.exposed = citace.pro.university
-rest.properties.exposed = citace.pro.allowed
```

Comment:
We need this properties. There are used in some places and UI will throw errors because they cannot load them.